spec/patron_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'yaml'
+require './lib/patron.rb'
+
+#collection = YAML.load_file('./lib/booksdb.yml')
+patron = YAML.load_file('./lib/patrondb.yml')
+
+describe Patron do
+  let(:library) {double('Library')}
+
+  it 'can view the list of books currently available in the library' do
+    expect(subject.available_books).not_to eq nil
+  end
+
+  it 'can view the list of the checked out books' do
+    expect(subject.checkedout_book_list).to eq patron.select { |obj| obj[:available] == false  }
+  end
```

Comment:
This method is testing that the `subject.checkedout_book_list` is `true` or `false`. Is that the intended behavior?